src/Controller/Component/UsersAuthComponent.php

```diff
@@ -114,6 +118,38 @@ protected function _initAuth()
     }
 
     /**
+     * Check Public Access
+     *
+     * ### Usage
+     *
+     * Your permssions config should return '*' or 'public' for role(s) allowed.
```

Comment:
It's not '' though, it's '*'.   You're actively turning on publicAcl & giving access to all roles with the asterisk.  And if we're using the english language as a convention :), "public" is as much of a role as any other role.  It's more confusing to to treat one group of people one way, and another group of people another way from a code flow perspective.  BTW, I'm working with Mark Story on an enhancement ticket and they seem to be leaning towards making "public" a part of the core as a user role in the AuthComponent.  

For reference here's the definition of role...
> the function assumed or part played by a person or thing in a particular situation.


FYI: I'm not trying to combative, and I know email/texts can come off that way.  Just sticking up for my position, and showing that I've done a lot of thinking about this issue.  I won't have hard feelings if you just close this ticket and don't merge it.